tests/src/test/java/alluxio/shell/command/SetTtlCommandIntegrationTest.java

```diff
@@ -97,11 +97,4 @@ public void setTtlWithInvalidOperationValue() throws Exception {
     Assert.assertEquals(-1,
         mFsShell.run("setTtl", "-action", "invalid", filePath, String.valueOf(ttl)));
   }
-
-  @Test
-  public void setTtlNegative() throws Exception {
```

Comment:
Before my changes: negative time values were interpreted by apache commons cli as invalid command line options( because they start with "-" ). However, The exeuction of the command continues and the value is later checked if it is negative or not and if it is negative we throw another exception with the "TTL value must be >= 0" which is redundant given that the user doesn't even also see the "TTL value must be >= 0" message.

Now, I added the if condition in line 69 in setTtlCommand file such that if there is a negative value and apache cli fails to parse the command then the execution will stop at this point and the negative value will not have any further effect on anything else. That's why we no longer need to verify or unit test entering negative time values.

I added a unit test for entering values with different time units.